compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -733,7 +734,9 @@ public static BType getReferredType(BType type) {
 
     public static void loadConstantValue(BType bType, Object constVal, MethodVisitor mv,
                                          JvmConstantsGen jvmConstantsGen) {
-
+        if (bType.tag == TypeTags.FINITE && ((BFiniteType) bType).getValueSpace().size() == 1) {
+            bType = ((BFiniteType) bType).getValueSpace().iterator().next().getBType();
```

Comment:
Do we have tests to cover this line

Reply:
This is added previously, but seems no usage of it. I will remove it